tests/library/test_ceph_osd.py

```diff
@@ -0,0 +1,267 @@
+from mock.mock import patch
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import json
+import os
+import pytest
+import ceph_osd
+
+fake_cluster = 'ceph'
+fake_container_binary = 'podman'
+fake_container_image = 'quay.ceph.io/ceph/daemon:latest'
+fake_id = '42'
+fake_ids = ['0', '7', '13']
+fake_user = 'client.admin'
+fake_keyring = '/etc/ceph/{}.{}.keyring'.format(fake_cluster, fake_user)
+invalid_state = 'foo'
+
+
+def set_module_args(args):
+    args = json.dumps({'ANSIBLE_MODULE_ARGS': args})
+    basic._ANSIBLE_ARGS = to_bytes(args)
+
+
+class AnsibleExitJson(Exception):
+    pass
+
+
+class AnsibleFailJson(Exception):
+    pass
+
+
+def exit_json(*args, **kwargs):
+    raise AnsibleExitJson(kwargs)
+
+
+def fail_json(*args, **kwargs):
+    raise AnsibleFailJson(kwargs)
```

Comment:
why don't you import this from `ca_test_common` maggle?

https://github.com/ceph/ceph-ansible/blob/master/tests/library/ca_test_common.py#L6-L29

Reply:
because I was not aware of that maggle